packages/firestore/test/unit/specs/describe_spec.ts

```diff
@@ -252,7 +252,7 @@ export function describeSpec(
     // Note: We use json-stable-stringify instead of JSON.stringify() to ensure
     // that the generated JSON does not produce diffs merely due to the order
     // of the keys in an object changing.
-    const output = stringify(specsInThisTest, {
+    const output = stringify.default(specsInThisTest, {
```

Comment:
Can you change the import to `import stringify from 'json-stable-stringify';` and leave here unchanged?

Reply:
Done